sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java

```diff
@@ -218,7 +218,7 @@ private static void validatePageMetadata(PageMetadata expectedPageInfo, PageMeta
      * Create a client builder with endpoint and API key credential.
      *
      * @param endpoint the given endpoint
-     * @param credential the given {@link FormRecognizerApiKeyCredential} credential
+     * @param credential the given {@link AzureKeyCredential} credential
```

Comment:
Just making sure that this little edit is all that needs to be changed in the key credential transition

Reply:
FormRecognizerApiKeyCredential is a deprecated type

Reply:
Sounds more like Isa was calling out if there were other places where the change needed to happen.
Looks like this was the only remaining though